### PR TITLE
chore: add config for pointing to Gitter chat

### PR DESCRIPTION
**Pull request by Midnighter**
This configuration enables an entry in the issue template picker that points to the Gitter channel instead.

### CODE REVIEW

**Comment by Midnighter** [comment]
`black` had an update last night and now annoyingly formats code differently. I don't think it makes sense to handle that in this PR.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true
```
Comment (euri10) [inline .github/ISSUE_TEMPLATE/config.yml:2]: maybe no blank issues dont you think ? it's either go to the chat, either use a template, or we'll end up again with issues partially filled

Comment (Midnighter) [reply .github/ISSUE_TEMPLATE/config.yml:2]: Good point, I must admit I copy and pasted this part.

**Comment by euri10** [comment]
> `black` had an update last night and now annoyingly formats code differently. I don't think it makes sense to handle that in this PR.

you're right I'm gonna PR a fix for this linting issue and  if you can rebase after that would be great so that stuff is seperated

**Comment by Midnighter** [comment]
> > `black` had an update last night and now annoyingly formats code differently. I don't think it makes sense to handle that in this PR.
> 
> you're right I'm gonna PR a fix for this linting issue and if you can rebase after that would be great so that stuff is seperated

I did it now here already...

**Review by euri10 — APPROVED** [review]

**Comment by euri10** [comment]
thanks @Midnighter !

**Comment by Midnighter** [comment]
Cheers :slightly_smiling_face:

**Pull request merged**